package-lock.json

```diff
@@ -2615,9 +2615,9 @@
       }
     },
     "typescript": {
-      "version": "2.4.2",
-      "resolved": "https://registry.npmjs.org/typescript/-/typescript-2.4.2.tgz",
-      "integrity": "sha1-+DlfhdRZJ2BnyYiqQYN6j4KHCEQ=",
+      "version": "2.6.1",
+      "resolved": "http://artifactory.es.ad.adp.com:8081/artifactory/api/npm/ohcm-npms/typescript/-/typescript-2.6.1.tgz",
```

Comment:
This is pointing to an internal registry

Reply:
Resolved. Thanks!